db/system_keyspace.cc

```diff
@@ -1586,6 +1586,9 @@ future<> system_keyspace::update_tokens(gms::inet_address ep, const std::unorder
     slogger.debug("INSERT INTO system.{} (peer, tokens) VALUES ({}, {})", PEERS, ep, tokens);
     auto set_type = set_type_impl::get_instance(utf8_type, true);
     co_await execute_cql(req, ep.addr(), make_set_value(set_type, prepare_tokens(tokens))).discard_result();
+    if (!_db.uses_schema_commitlog()) {
```

Comment:
nit: Maybe keep the check in force_blocking_flush() itself (under relevant bool_class argument if needed)?

Reply:
We could probably have `force_blocking_flush_if_no_schema_commitlog`, but the number of letters in the name makes it hardly simpler than the plain `if`s ) I also don't see how something like `force_blocking_flush_conditionally(cond)` would improve the code here.